src/Stryker.Core/Stryker.Core.UnitTest/Initialisation/AssemblyReferenceResolverTests.cs

```diff
@@ -39,16 +39,16 @@ public void ShouldAnalyzeProjectDependencies()
             var result = target.ResolveReferences(Path.GetDirectoryName(project), Path.GetFileName(project), "ExampleProject").ToList();
 
             processExecutorMock.Verify(x => x.Start(
-                Path.GetDirectoryName(project), 
-                "dotnet", "msbuild ExampleProject.Test.csproj /nologo /t:PrintReferences", 
+                Path.GetDirectoryName(project),
+                "dotnet", "msbuild ExampleProject.Test.csproj /nologo /t:PrintReferences",
                 null,
-                It.IsAny<int>()), 
+                It.IsAny<int>()),
                 Times.Once);
             foundReferences.ShouldBeSubsetOf(new List<string>()
             {
-                Path.Combine($@"C:{Path.DirectorySeparatorChar}Repos", "ExampleProject", "ExampleProject", "bin", "Debug", "netcoreapp2.0", "ExampleProject.Contracts.dll"),
-                Path.Combine($@"C:{Path.DirectorySeparatorChar}Program Files", "dotnet", "sdk", "NuGetFallbackFolder", "microsoft.netcore.app", "2.0.0", "ref", "netcoreapp2.0", "Microsoft.CSharp.dll"),
-                Path.Combine($@"C:{Path.DirectorySeparatorChar}Program Files", "dotnet", "sdk", "NuGetFallbackFolder", "microsoft.aspnetcore.mvc.formatters.json", "2.0.2", "lib", "netstandard2.0", "Microsoft.AspNetCore.Mvc.Formatters.Json.dll")
+                Path.Combine("C:", "Repos", "ExampleProject", "ExampleProject", "bin", "Debug", "netcoreapp2.0", "ExampleProject.Contracts.dll"),
```

Comment:
This unit test is failing on windows because C:Repos\ is expected but the actual file path value on windows is C:\Repos\